### PR TITLE
TASK: Adjust to "serializable commands"

### PR DESCRIPTION
**Pull request by bwaidelich**
Related: https://github.com/neos/neos-development-collection/issues/5353
Related: https://github.com/neos/neos-development-collection/pull/5348

### CODE REVIEW

File: Classes/Infrastructure/ContentRepository/ConflictsFactory.php [hunk 1]
```diff
@@ -130,67 +124,42 @@ private function createConflictFromCommandThatFailedDuringRebase(
...
-            $command instanceof TagSubtree,
-            $command instanceof DisableNodeAggregate,
-            $command instanceof UntagSubtree,
-            $command instanceof EnableNodeAggregate,
-            $command instanceof RemoveNodeAggregate =>
-                $command->coveredDimensionSpacePoint,
-            $command instanceof ChangeNodeAggregateType =>
+        $dimensionSpacePoint = match ($event::class) {
+            NodeAggregateWasMoved::class =>
+                $event->succeedingSiblingsForCoverage->toDimensionSpacePointSet()->getFirst(),
```
Comment (mhsdesign) [inline Classes/Infrastructure/ContentRepository/ConflictsFactory.php]: todo this is maybe cheesy? We look into the `succeedingSiblingsForCoverage` ???

File: Classes/Infrastructure/ContentRepository/ConflictsFactory.php [hunk 2]
```diff
@@ -130,67 +124,42 @@ private function createConflictFromCommandThatFailedDuringRebase(
...
+            NodePropertiesWereSet::class,
+            NodeAggregateWithNodeWasCreated::class =>
+                $event->originDimensionSpacePoint->toDimensionSpacePoint(),
+            NodeReferencesWereSet::class =>
+                $event->affectedSourceOriginDimensionSpacePoints->toDimensionSpacePointSet()->getFirst(),
```
Comment (mhsdesign) [inline Classes/Infrastructure/ContentRepository/ConflictsFactory.php]: not ideal that there is only a set here and not a `originDimensionSpacePoint` like in `SetSerializedNodeProperties`?

**Review by mhsdesign — APPROVED** [review]
But it seems to work so far despite the not perfect `getFirst` things ;)

File: Classes/Infrastructure/ContentRepository/ConflictsFactory.php (hunk 1 above)
Comment (bwaidelich) [reply Classes/Infrastructure/ContentRepository/ConflictsFactory.php]: Nasty.. I created an issue for that https://github.com/neos/neos-development-collection/issues/5360

**Pull request merged**